src/labels/CustomLabels.labels

```diff
@@ -128,6 +128,30 @@
         <shortDescription>Preferred Email Required Error</shortDescription>
         <value>Tell us which email is preferred.</value>
     </labels>
+    <labels>
```

Comment:
If you do a full metadata retrieve of the metadata from the org (do it when you don't have any pending work to commit!), there should be updates to our translation files for these labels.

Reply:
I think this still needs to be done?

Reply:
correct, I left it for after the review.